runtimes/bun-1.0/src/server.ts

```diff
@@ -38,6 +38,11 @@ const action = async (logger: Logger, request: any) => {
     headers[header.toLowerCase()] = request.headers.get(header);
   });
 
+  const enforcedHeaders = JSON.parse(Bun.env["OPEN_RUNTIMES_HEADERS"] ? Bun.env["OPEN_RUNTIMES_HEADERS"] : '{}');
+  for(const header in enforcedHeaders) {
```

Comment:
```suggestion
  for (const header in enforcedHeaders) {
```
can we run some linter to fix these ?

Reply:
I can make myself task to setup linter (+test) right after v4, right after merging runtime versions together. For now I would like to avoid to keep this PRs simple. Linter never ran here as far as I can remember.